spaceship/lib/spaceship/tunes/iap.rb

```diff
@@ -0,0 +1,112 @@
+require 'spaceship/tunes/iap_list'
+require 'spaceship/tunes/iap_detail'
+require 'spaceship/tunes/iap_status'
+require 'spaceship/tunes/iap_type'
+require 'spaceship/tunes/iap_family_list'
+require 'spaceship/tunes/iap_families'
+require 'spaceship/tunes/iap_family_details'
+require 'spaceship/tunes/iap_families'
+
+module Spaceship
+  module Tunes
+    class IAP < TunesBase
+      # @return (Spaceship::Tunes::Application) A reference to the application
+      attr_accessor :application
+
+      class << self
+        def factory(attrs)
+          return self.new(attrs)
+        end
+      end
+
+      # @return (Spaceship::Tunes::IAPFamilies) A reference to the familie list
+      def families
+        attrs = {}
+        attrs[:application] = self.application
+        Tunes::IAPFamilies.factory(attrs)
+      end
+
+      # Creates a new In-App-Purchese on iTunes Connect
```

Comment:
it throws a exception `Spaceship::TunesClient::ITunesConnectError` - as it calls the `handle_itc_response` like most of the other tunes_client stuff. added a comment for it.